src/issuance.rs

```diff
@@ -1262,13 +1258,13 @@ mod tests {
 
     #[test]
     fn issue_bundle_verify_fail_incorrect_ik() {
-        let asset_description = "Asset";
+        let asset_description: Vec<u8> = "Asset".into();
```

Comment:
yes that works but what we really need is to specify exactly the conversion:
```rust
let asset_description = b"Asset".to_vec();
```
Please move to this syntax every time you convert `"string"` to `Vec<u8>`

Reply:
Understood, moved to this format in all places